airflow/providers/google/cloud/operators/dataflow.py

```diff
@@ -1018,12 +1018,12 @@ class DataflowCreatePythonJobOperator(GoogleCloudBaseOperator):
     """
     Launching Cloud Dataflow jobs written in python.
 
-     Note that both dataflow_default_options and options will be merged to specify pipeline
```

Comment:
There was a [redacted] [redacted] [redacted] leading space in that line which was causing the indent error. 🤦    That was like playing "find the misplaced semicolon" LOL

Reply:
BEEN THERE DONE THAT.